resources/helm/overlays/istio_cni/templates/configmap-v2.3.yaml

```diff
@@ -0,0 +1,26 @@
+{{ if .Values.cni.enabled }}
+{{- if and (has "v2.3" .Values.cni.supportedReleases) (eq .Values.cni.instanceVersion "v2.3") }}
+# This ConfigMap is used to configure a self-hosted Istio CNI installation.
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: istio-cni-config-v2-3
+  namespace: {{ .Release.Namespace }}
+data:
+  # The CNI network configuration to add to the plugin chain on each node.  The special
+  # values in this config will be automatically populated.
+  cni_network_config_v2_3: |-
```

Comment:
We should probably change this to just `cni_network_config`, since it's the only entry in this ConfigMap, but we can do this in a follow-up PR.

Reply:
I rebased the CM data entry name change in this PR. The only place we set that network config name is in:
https://github.com/maistra/istio-operator/pull/1017/files#diff-fb85c6f4f7143e3ebb7cf98f29325df00f2945710262a52c73b64d790c12ed53R64